tests/components/axion_dmx/test_light.py

```diff
@@ -0,0 +1,134 @@
+"""Test the Axion DMX light."""
+
+import tempfile
+import unittest
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from homeassistant.components.axion_dmx.light import AxionDMXLight
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_HS_COLOR,
+    ATTR_RGBW_COLOR,
+    ATTR_RGBWW_COLOR,
+    ColorMode,
+)
+from homeassistant.core import HomeAssistant
+
+
+class TestAxionDMXLight(unittest.TestCase):
```

Comment:
Let's not write test cases like this. Please create tests here that look like the `test_config_flow` ones